lib/paramedic.js

```diff
@@ -244,6 +244,20 @@ ParamedicRunner.prototype.setPermissions = function () {
             paramediciOSPermissions.updatePermissions(applicationsToGrantPermission);
         }
     }
+
+    if(this.config.getPlatformId() === util.ANDROID) {
```

Comment:
With the permissions granted explicitly, this would probably not be needed any more, correct?
https://github.com/apache/cordova-paramedic/blob/ff6b2fa00b2e5b11907f2e7d0296f7b35feb4a04/lib/appium/helpers/wdHelper.js#L262-L295
Which would be good - because it is a source for errors and timeouts :/